configs/ballerina.json

```diff
@@ -1,52 +1,142 @@
 {
-  "index_name": "ballerina",
-  "start_urls": [
-    {
-      "url": "https://ballerina.io",
-      "tags": ["ballerina"],
-      "extra_attributes": {
-        "site": ["ballerina"]
-      },
-      "page_rank": 1
+    "index_name": "ballerina",
+    "start_urls": [
+        {
+            "url": "https://ballerina.io",
+            "tags": [
+                "ballerina"
+            ],
+            "extra_attributes": {
+                "site": [
+                    "ballerina"
+                ]
+            },
+            "page_rank": 1
+        },
+        {
+            "url": "https://lib.ballerina.io",
+            "page_rank": 2,
+            "selectors_key": "swanlake_api_docs",
+            "tags": [
+                "swanlake_api_docs"
+            ],
+            "extra_attributes": {
+                "site": [
+                    "swanlake_api_docs"
+                ]
+            }
+        },
+        {
+            "url": "https://blog.ballerina.io",
+            "page_rank": 3,
+            "selectors_key": "ballerina_blog",
+            "tags": [
+                "ballerina_blog"
+            ],
+            "extra_attributes": {
+                "site": [
+                    "ballerina_blog"
+                ]
+            }
+        },
+        {
+            "url": "https://central.ballerina.io",
+            "page_rank": 4,
+            "selectors_key": "ballerina_central",
+            "tags": [
+                "ballerina_central"
+            ],
+            "extra_attributes": {
+                "site": [
+                    "ballerina_central"
+                ]
+            }
+        },
+        {
+            "url": "https://play.ballerina.io",
+            "page_rank": 5,
+            "selectors_key": "ballerina_playground",
+            "tags": [
+                "ballerina_playground"
+            ],
+            "extra_attributes": {
+                "site": [
+                    "ballerina_playground"
+                ]
+            }
+        }
```

Comment:
As this is interactive content, it should exists in the config since there's nothing to crawl on this page. Plus, when searching for `playground`, it already redirects to this page: https://ballerina.io/learn/user-guide/getting-started/setting-up-ballerina/#trying-ballerina-in-the-playground

Reply:
@Clément Vannicatte You know what best. Please add that.

Reply:
```suggestion
```